examples/example01/README.md

```diff
@@ -0,0 +1,137 @@
+# Steps to run this example, i.e. GaAs wannierization
+
+ Two possibilities are provided:\
+ A) Run a full calculation from scratch QE+Wannier90 (from step 1 to step 5)\
+ B) Run only the Wannier90 part (step 5 only)
+
+ We use in the following the  verdi command-line interface (CLI). Reminder for simple command in the updated AiiDA version:
+1. workon aiida  (to enter the AiiDA cirtual environment)
```

Comment:
Small typo: _virtual_ environment. Maybe we should make it clear that this is not the same for all kinds of environment managers.

Suggestion:
"Activate your virtual environment, e.g. with ``workon aiida``"